UPGRADE.md

```diff
@@ -646,7 +646,7 @@ const UserCreateToolbar = ({ permissions, ...props }) =>
                 label="user.action.save_and_add"
                 redirect={false}
                 submitOnEnter={false}
-                raised={false}
+                variant="flat"
```

Comment:
This is the default, so it's not necessary. See https://material-ui-next.com/demos/buttons/.

Same for all the other flat buttons.

Reply:
That would be true if it was a simple Mui Button. However, our `SaveButton` set the variant to `raised` by default

Reply:
meh

Reply:
See marmelab/admin-on-rest/pull/1486/files/a074016fcd20b1a9f2a2edd25ac58b3437a40c57#diff-3affbe736d22fedadd658b6ffd6cb70f